apps/dokploy/i18n/InternationalizationKit/interface/locales.d.ts

```diff
@@ -0,0 +1,662 @@
+/* Autogenerated file. Do not edit manually. */
+
+interface Locales {
+	/**
+	 * Dokploy
+	 */
+	'PAGE.register.Dokploy': string;
+	/**
+	 * Setup the server
+	 */
+	'PAGE.register.setupServer': string;
+	/**
```

Comment:
How did you generate this file? do you have command?

Reply:
> How did you generate this file? do you have command?

It is necessary to use `smartLocale.js` to generate `locales.d.ts`, and configure `SmartLocalePlugin` in `next.config.mjs`.

```javascript
		config.plugins.push(new SmartLocalePlugin({
			inputDirectory: "./i18n/locales/",
			outputDirectory: "./i18n/InternationalizationKit/interface/",
			outputImportDirectory: "./i18n/",
			defaultLocale: "en-us",
			fileType: "json",
		}));

Reply:
So they generate automatically?

Reply:
That's right, it will execute once when the development server starts.